src/Controller/BlobController.php

```diff
@@ -76,7 +76,7 @@ public function connect(Application $app)
             list($branch, $file) = $app['util.routing']
                 ->parseCommitishPathParam($commitishPath, $repo);
 
-            $filePatchesLog = $repository->getCommitsLogPatch($file);
+            $filePatchesLog = $repository->getCommitsLogPatch($file,$branch);
```

Comment:
If we want to follow PSR2 there should be a space between the comma and the $branch variable.